test/parallel/test-runner-mocking.js

```diff
@@ -534,6 +534,69 @@ test('mocks a setter', (t) => {
   assert.strictEqual(obj.prop, 65);
 });
 
+test('mocks a getter with syntax sugar', (t) => {
+  const obj = {
+    prop: 5,
+    get method() {
+      return this.prop;
+    },
+  };
+
+  function mockMethod() {
+    return this.prop - 1;
+  }
+  const getter = t.mock.getter(obj, 'method', mockMethod);
```

Comment:
@cjihrig Thank you for your suggestions. 
If `getter` cannot be set to a different value, I think that validation do not work for `options` in method().
Do we implement validation for `options` in `getter()` and `setter()` ?

Reply:
> If getter cannot be set to a different value, I think that validation do not work for options in method().

There is validation, but if you set the `getter` option to `false`, it would pass validation, but the `getter()` method would not make sense.

> Do we implement validation for options in getter() and setter() ?

I would be OK with validating the `getter` or `setter` option to ensure that they are not set to any value besides `true` and deferring all other validation to `method()`.